pat/methods.go

```diff
@@ -1,15 +1,12 @@
 package pat
 
-import (
-	"goji.io"
-	"goji.io/pattern"
-)
+import "goji.io"
 
 /*
 Delete returns a Pat route that only matches the DELETE HTTP method.
 */
 func Delete(pat string) goji.Pattern {
```

Comment:
We should return `*Pattern` here (which, among other things, will make godoc look more sensible)